ansible/group_vars/all/defaults.yaml

```
@@ -21,8 +21,15 @@ crictl_url: https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ c
 crictl_sha256: 44d5f550ef3f41f9b53155906e0229ffdbee4b19452b4df540265e29572b899c
 containerd_cri_socket: /run/containerd/containerd.sock
 systemd_prefix: /usr/lib/systemd/site-packages
```

Instead of having flatcar specific vars.

You can dos something like for `systemd_prefix` and `sysusrlocal_prefix`

```suggestion
systemd_prefix: "{{ '/etc/systemd' if ansible_os_family == "Flatcar" else '/usr/lib/systemd/site-packages' }}" 
```

And then you can remove the Flatcar specific tasks in `ansible/roles/packages/tasks/url.yaml`.

---

I'll try this out in a little bit later tonight. I'm not sure this will work because i dont think all of the ansible_os_family information will be set or not at this point in the run